docs/components/movement-sensor/adxl345.md

```diff
@@ -32,7 +32,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
 ```json {class="line-numbers linkable-line-numbers"}
 {
   "board": "<your-board-name>",
```

Comment:
I'd order these the same as the attribute descriptions below. If you like my other suggestion, move this line down below "use_alternate_i2c_address" and above "tap". That way, the required attributes (i.e., only "i2c_bus" any more) are at the top.

Reply:
Done